### DISCUSSION TITLE
Where to type the Konami code to get the cat blocks?

### DISCUSSION THREAD

**Discussion by Genius398** [ScratchAddons/ScratchAddons#1648] [Help] [answered]
Where to type the Konami code to get the cat blocks?

**Comment by pufferfish101007** [comment]
The main settings page.

**Comment by Genius398** [comment]
> The main settings page.

How?

**Comment by Genius398** [comment]
> The main settings page.

Can I do it now?

What is the code?

**Comment by mxmou** [comment, marked as answer]
You just have to be on the settings page and type [the code](https://en.wikipedia.org/wiki/Konami_Code).

**Comment by Genius398** [comment]
> You just have to be on the settings page and type [the code](https://en.wikipedia.org/wiki/Konami_Code).

Thanks!

**Comment by Basic8** [comment]
I know this is answered but, you have to do up up down down left right left right B A on the page.
